kv/task.go

```diff
@@ -971,13 +988,16 @@ func (s *Service) findRuns(ctx context.Context, tx Tx, filter influxdb.RunFilter
 		return nil, 0, err
 	}
 	for _, run := range currentlyRunning {
-		runs = append(runs, run)
+		if run.ScheduledFor.After(parsedFilterAfterTime) && run.ScheduledFor.Before(parsedFilterBeforeTime) {
+			runs = append(runs, run)
+		}
 		if len(runs) >= filter.Limit {
 			return runs, len(runs), nil
 		}
 	}
 
 	return runs, len(runs), nil
+
```

Comment:
nit: :scissors: ime, it's more beneficial to add spacing inside the function between logical blocks, than to add a space as the first or last line in a function. Julia Ferraioli's "Writing Accessible Go" is a great resource to look at for more accessibility tips.